libs/utils/utils.go

```diff
@@ -131,3 +132,21 @@ func InterfaceSliceToStringSlice(originalSlice []interface{}) []string {
 	}
 	return newSlice
 }
+
+// RemoveDuplicatesFromSlice ...
+func RemoveDuplicatesFromSlice(list []string) string {
+	cleaned := []string{}
+	m := make(map[string]bool)
+
+	for _, item := range list {
+		if item == "" {
+			continue
+		}
+		if _, ok := m[item]; !ok {
+			m[item] = true
+			cleaned = append(cleaned, strings.TrimSpace(item))
+		}
+	}
+
+	return strings.Join(cleaned[:], ", ")
```

Comment:
Since we are now making this a utility, we should return a list here instead of a string joining the items delimited by a delimiter (comma in this case).

Reply:
Done the changes.